docs/static/monitoring/monitoring-ea-intro.asciidoc

```diff
@@ -13,13 +13,15 @@ integrations for new data sources, security protections, and more.
 
 You can use {agent} to collect monitoring data for:
 
+* Serverless dashboards
```

Comment:
@karenzone Note to self:  Placeholder to be fleshed out with links and descriptive text